frontend/src/utils/funtioncs.ts

```diff
@@ -0,0 +1,7 @@
+export function capitalizeFirstLetter(string: string) {
+    const splitStr: string[] = string.toLowerCase().split(' ');
+    for (let i = 0; i < splitStr.length; i++) {
+        splitStr[i] = splitStr[i].charAt(0).toUpperCase() + splitStr[i].substring(1);
+    }
+    return splitStr.join(' ');
+}
```

Comment:
U can achieve the same thing in a simpler way:

let stringEx1 = "example"

stringEx1 = stringEx1[0].toUpperCase() + stringEx1.slice(1)

console.log(stringEx1) => Example